tslint.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
There is an indent rule that we could turn on, but it wouldn't allow us to easily merge all the other open pull requests.  https://palantir.github.io/tslint/rules/indent/